.github/stale.yml

```diff
@@ -0,0 +1,16 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 14
```

Comment:
Looks like it is going to close the issue if no activity happens. But I think we should add a label i.e. `status/state` to the issue and then  we will take manual action.

Reply:
As per the stale probot documentation there is no mention of how someone can skip the closing part. So thats an issue as if the issue gets closed then I think non-memebers can not even re-open the issue, isn't it?

Reply:
> So thats an issue as if the issue gets closed then I think non-members of the project can not even re-open the issue, isn't it?

As long as someone changes anything on the issue with 14 days, it won't get closed. There is really no different to what a human would do. 

I think we can give it a trial run. I can watch it as part of my archeologist week. If we don't like it, we change the settings or disable it completely. We can even remove all sorts of traces the bot left.

Reply:
Sure, lets give it a try.